src/main/UserActivityMonitor.js

```diff
@@ -77,51 +65,52 @@ export default class UserActivityMonitor extends EventEmitter {
       } catch (err) {
         console.log('Error getting system idle time:', err);
       }
-    }, this.config.internalUpdateFrequencyMs);
+    }, this.config.updateFrequencyMs);
   }
 
   /**
    * Stop monitoring system events and idle time
    */
   stopMonitoring() {
-    electron.powerMonitor.off('suspend', this.handleSystemGoingAway);
-    electron.powerMonitor.off('resume', this.handleSystemComingBack);
-    electron.powerMonitor.off('lock-screen', this.handleSystemGoingAway);
-    electron.powerMonitor.off('unlock-screen', this.handleSystemComingBack);
-
     clearInterval(this.systemIdleTimeIntervalID);
   }
 
   /**
-   * Updates internal idle time properties and conditionally triggers updates to user activity status
+   * Updates internal idle time and sets internal user activity state
    *
    * @param {integer} idleTime
    * @private
    */
   updateIdleTime(idleTime) {
     this.idleTime = idleTime;
-
-    if (this.idleTime > this.config.activityTimeoutSec) {
-      this.updateUserActivityStatus(false);
-    } else if (!this.forceInactive && this.idleTime < this.config.activityTimeoutSec) {
-      this.updateUserActivityStatus(true);
+    if (idleTime > this.config.inactiveThresholdSec) {
+      this.setActivityState(false);
+    } else {
+      this.setActivityState(true);
     }
   }
 
   /**
-   * Updates user activity status if changed and triggers a status update
+   * Updates user active state and conditionally triggers a status update
    *
    * @param {boolean} isActive
    * @param {boolean} isSystemEvent – indicates whether the update was triggered by a system event (log in/out, screesaver on/off etc)
    * @private
    */
-  updateUserActivityStatus(isActive = false, isSystemEvent = false) {
+  setActivityState(isActive = false, isSystemEvent = false) {
+    this.isActive = isActive;
+
+    if (isSystemEvent) {
+      this.sendStatusUpdate(true);
```

Comment:
Do system events still occur? If so, should we be returning early here, or do we still want to fall through and set `lastSetActive` and potentially invoke `sendStatusUpdate` a second time?

Reply:
Gah, I'm seeing double ... thanks for catching that! So, for v4.3, system events have been removed, but I left the infrastructure in for later. I could remove it all the way back to just before transitioning to the webapp. The webapp is expecting a boolean for `manual`, which is set based on `isSystemEvent` from this file ... I could hardcode to `false` for now. Thoughts?